router/core/graphql_handler.go

```diff
@@ -375,9 +375,16 @@ func addErrorToSpan(ctx context.Context, err error) {
 	rtrace.AttachErrToSpan(span, err)
 }
 
-func propagateSubgraphErrors(ctx *resolve.Context) {
+func propagateSubgraphErrors(ctx *resolve.Context, requestLogger *zap.Logger) {
+	// Get subgraph errors
 	err := ctx.SubgraphErrors()
+	// Add error to span
 	addErrorToSpan(ctx.Context(), err)
+	// Check if there are subgraph errors
+	if err != nil {
+		// Log errors to know that errors occurred
+		requestLogger.Error("error detected in subgraphs", zap.Error(ctx.SubgraphErrors()))
```

Comment:
No need to call it again. You can use the err from above :smile:

Reply:
🤦🤦
I'm sorry... 3 lines of code, 1 fail at each line... I will patch this today

Reply:
I've edited the code to have a quick return like done in other part of the code.